src/tests/baseservices/RuntimeConfiguration/TestConfig.csproj

```diff
@@ -1,15 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <!-- This test provides no interesting scenarios for GCStress -->
-    <GCStressIncompatible>true</GCStressIncompatible>
-    <UnloadabilityIncompatible>true</UnloadabilityIncompatible>
-    <DisableProjectBuild Condition="'$(RuntimeFlavor)' == 'Mono'">true</DisableProjectBuild>
+    <!-- This is a separate app launched by the actual test in TestConfigTester.csproj -->
+    <RequiresProcessIsolation>true</RequiresProcessIsolation>
+    <CLRTestKind>BuildOnly</CLRTestKind>
+    <NoWarn>$(NoWarn);XUW1001</NoWarn>
```

Comment:
Is setting `ReferenceXUnitWrapperGenerator` to `false` appropriate here?

It would remove the need for the `NoWarn`. The other warnings might not be relevant anyway, and I think we don't want BuildAsStandalone trying to build a simple wrapper for this. So I somewhat think the answer is "yes", but I don't feel certain about it.

Reply:
Removed. good point.

Reply:
(Removed NoWarn and added ReferenceXUnitWrapperGenerator=false to be precise.)